### DISCUSSION TITLE
ImmortalWrt 在线构建服务 使用说明

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/blog#18] [Comments]
https://blog.imouto.in/post/openwrt/2023/immortalwrt-asu-usage/

**Comment by maodeyu180** [comment]
佬，这个博客怎么搭建啊  直接fork工程 就可以吗

**Comment by gegj** [comment]
https://asu-2.kyarucloud.moe/ 这个挂了，现在没法定制固件了？

**Comment by cocolight** [comment]
感谢大佬分享
